crypto_commons/generic.py

```diff
@@ -196,7 +220,11 @@ def xor_hex(t1, t2):
 
 def is_printable(data):
     import string
+    printable = set(string.printable)
```

Comment:
Maybe simply `return len(set(data).intersection(set(string.printable))) > 0` ?

Reply:
Or rather `len(set(data).difference(printable)) == 0`, your version will detect any string with a printable byte. Anyway, will be fixed.

Reply:
This is what I was thinking about :)